packages/ccip-svm/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "tsc ",
     "dev": "tsc -w",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "vitest"
```

Comment:
Let's stick to jest to keep tooling consistent with ccip-js.
- jest for testing
- eslint configs
- prettier configs
that way we get consistent envs across all our dev machines, and we dont get formatting /linting diffs on autosave etc.

Reply:
yes ccip-js is being updated for eslint - thanks!